### PR TITLE
LibWeb: Implement the flex order CSS property

### PR DESCRIPTION
**Pull request by Enverbalalic**
Adds support for the flex order property and a test page for it
on the browser welcome page.

### CODE REVIEW

File: Userland/Libraries/LibWeb/CSS/Properties.json [hunk 1]
```diff
@@ -689,6 +689,13 @@
       "number"
     ]
   },
+  "order": {
+    "inherited": false,
+    "initial": "0",
+    "valid-types": [
+      "number"
```
Comment (AtkinsSJ) [inline Userland/Libraries/LibWeb/CSS/Properties.json:696]: This is `<integer>` in the spec, so it should be "integer" here.

File: Userland/Libraries/LibWeb/CSS/StyleProperties.cpp [hunk 2]
```diff
@@ -263,6 +263,14 @@ float StyleProperties::flex_shrink() const
     return value.value()->to_number();
 }
 
+int StyleProperties::order() const
+{
+    auto value = property(CSS::PropertyID::Order);
+    if (!value.has_value() || !value.value()->has_number())
+        return 0;
+    return value.value()->to_number();
```
Comment (AtkinsSJ) [inline Userland/Libraries/LibWeb/CSS/StyleProperties.cpp:271]: As with Properties.json, this code wants to be working with integers - `has_integer()` and `to_integer()`.

**Review by AtkinsSJ — CHANGES_REQUESTED** [review]
Very cool that you're working on this! I won't review the code that does the actual ordering, but I've spotted a couple of little issues.

**Comment by Enverbalalic** [comment]
Ah, I assumed this was the JS meaning of number. I'll adjust, thanks.

**Pull request merged**